move/gas_service/sources/gas_service.move

```diff
@@ -71,6 +81,34 @@ fun init(ctx: &mut TxContext) {
     );
 }
 
+// ------
+// Macros
+// ------
+// We allow unused functions here, vbecause if in the future anyone wants to access the balance mutliple times
+// they should load it with data first. The getter `balance!()` is here for tests and completeness.
+#[allow(unused_function)]
+macro fun data($self: &GasService): &GasServiceDataV0 {
+    let gas_service = $self;
+    gas_service.inner.load_value<GasServiceDataV0>()
+}
+
+#[allow(unused_function)]
+macro fun data_mut($self: &GasService): &mut GasServiceDataV0 {
+    let gas_service = $self;
+    gas_service.inner.load_value_mut<GasServiceDataV0>()
+}
+
+#[allow(unused_function)]
```

Comment:
can you mark this as `test_only`?

Reply:
you can file a request to mysten team if it's not supported. And do `self.data!().balance()` in tests for now (want to avoid becoming ok with adding lint exceptions everywhere)